aldryn_forms/boilerplates/bootstrap3/templates/aldryn_forms/fields/textareafield.html

```diff
@@ -3,7 +3,7 @@
 {% if field %}
     <div class="form-group{% if field.field.required %} form-required{% endif %}{% if field.errors %} has-error{% endif %}">
         {% block form_field_label %}
-            <label for="{{ field.auto_id }}" class="sr-only">
+            <label for="{{ field.auto_id }}" {% if not field.label %}class="sr-only"{% endif %}>
```

Comment:
and now it renders an empty space, thats correct:

``<label for="{{ field.auto_id }}"{% if not field.label %} class="sr-only"{% endif %}>``